package.json

```diff
@@ -41,14 +42,20 @@
       }
     }
   },
-  "workspaces": [
-    "docs",
-    "angular-docs",
-    "environments",
-    "examples/*",
-    "packages/*",
-    "packages/angular/projects/ui-angular"
-  ],
+  "workspaces": {
+    "packages": [
+      "docs",
+      "angular-docs",
+      "environments",
+      "examples/*",
+      "packages/*",
+      "packages/angular/projects/ui-angular"
+    ],
+    "nohoist": [
+      "**/react-native",
+      "**/react-native/**"
```

Comment:
Why do we have to make it `nohoist`?

Reply:
React Native assumes dependent modules reside inside the app's `node_modules` during installation and runtime, which results in conflicts with the hoist scheme of yarn workspaces

Reply:
Ouch, that's annoying then :(

Reply:
Monorepo painz